### PR TITLE
[IMP] style: add vertical text align option

### PR DESCRIPTION
**Pull request by laa-odoo**
## Description:

This commit makes it possible to vertically align text.
These options have been added to the format menu.
Thus, it is now possible to align text:
- to the top
- to the middle
- to the bottom

Also, the options to horizontally align the text (present in the tool bar)
have also been added to the format menu.

Odoo task ID : [2886061](https://www.odoo.com/web#id=2886061&action=333&active_id=2328&model=project.task&view_type=form&cids=1&menu_id=4720)

## review checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_lt("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

### CODE REVIEW

File: src/plugins/ui/renderer.ts [hunk 1]
```diff
@@ -287,14 +294,15 @@ export class RendererPlugin extends UIPlugin {
           ctx.clip();
         }
 
-        const brokenLineNumber = box.content.multiLineText.length;
-        const size = computeTextFontSizeInPixels(style);
-        const contentHeight =
-          brokenLineNumber * (size + MIN_CELL_TEXT_MARGIN) - MIN_CELL_TEXT_MARGIN;
-        let brokenLineY = y - contentHeight / 2;
+        // compute vertical align start point parameter:
+        const textLineSize = computeTextFontSizeInPixels(style);
```
Comment (LucasLefevre) [inline src/plugins/ui/renderer.ts:298]:
```suggestion
        const textLineHeight = computeTextFontSizeInPixels(style);
```

**Review by LucasLefevre — COMMENTED** [review]
mostly ok :)
All those renderer things are not easy though and it's not getting better with time

File: src/plugins/ui/renderer.ts [hunk 2]
```diff
@@ -259,26 +260,32 @@ export class RendererPlugin extends UIPlugin {
       if (box.content) {
         const style = box.style || {};
         const align = box.content.align || "left";
+
+        // compute font and textColor
```
[the remark this answers is not in the document]
Comment (laa-odoo) [reply src/plugins/ui/renderer.ts]: A work to do on all the functions of this file:
https://www.odoo.com/web#id=3085055&menu_id=4720&cids=1&action=333&active_id=2328&model=project.task&view_type=form

**Review by pro-odoo — APPROVED** [review]
robodoo r+

**Pull request closed without merging**